sysutils/flashrom/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                flashrom
+version             1.0
+categories          sysutils
+platforms           darwin
+license             GPL-2
+maintainers         ra1nb0w openmaintainer
```

Comment:
You need to prepend your Github handle with @. Can you also add your obfuscated email address? So the maintainers line will be like this:

`maintainers         {@ra1nb0w domain.com:name} openmaintainer`

Reply:
not too much obfuscated if always the same ;-)